functions/index.js

```diff
@@ -0,0 +1,24 @@
+const functions = require('firebase-functions');
+const fetch = require('node-fetch');
+const HONESTY_STORE_LOGO = 'https://honesty.store/assets/android/icon@MDPI.png';
+const BOT_USERNAME = 'honesty.store';
+
+exports.sendSlackMessage = functions.https.onCall(data => {
+  const token = functions.config().react_app.slack_token;
+
+  const result = fetch(
```

Comment:
Can we return the fetch promise directly since we're assigning and then returing?

e.g.

```
return fetch(...)
  .then(response => response.json())
  .catch(...)
```